app/experimenter/experiments/constants/nimbus.py

```diff
@@ -156,25 +157,115 @@ class ApplicationConfig:
 )
 
 
+class Status(models.TextChoices):
+    DRAFT = "Draft"
+    PREVIEW = "Preview"
+    LIVE = "Live"
+    COMPLETE = "Complete"
+
+
+class PublishStatus(models.TextChoices):
+    IDLE = "Idle"
+    REVIEW = "Review"
+    APPROVED = "Approved"
+    WAITING = "Waiting"
+
+
+class LifecycleStates(Enum):
+    DRAFT_IDLE = dict(
+        status=Status.DRAFT,
+        publish_status=PublishStatus.IDLE,
+    )
+    PREVIEW_IDLE = dict(
+        status=Status.PREVIEW,
+        publish_status=PublishStatus.IDLE,
+    )
+    DRAFT_REVIEW = dict(
+        status=Status.DRAFT,
+        publish_status=PublishStatus.REVIEW,
+    )
+    DRAFT_APPROVED = dict(
+        status=Status.DRAFT,
+        publish_status=PublishStatus.APPROVED,
+    )
+    DRAFT_WAITING = dict(
+        status=Status.DRAFT,
+        publish_status=PublishStatus.WAITING,
+    )
+    LIVE_IDLE = dict(
+        status=Status.LIVE,
+        publish_status=PublishStatus.IDLE,
+    )
+    PAUSE_QUEUED = dict(
+        status=Status.LIVE,
+        is_paused=False,
+    )
```

Comment:
This kind of stands out from the others that are named as the pair of status/publishstatus, maybe it's unavoidable, but it stands out a bit to me 🤷‍♀️

Reply:
Yeah, I thought maybe this one should also have `publishStatus`, but I lifted it as-is from the `IS_PAUSE_QUEUED` filter and didn't want to mess with it further